app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -107,14 +113,62 @@
       <% end %>
     <% end %>
 
-    <div id="wrapper" class="<%= "govuk-width-container" unless full_width %>">
-      <%= yield :before_content %>
-      <main class="govuk-main-wrapper" id="content">
-        <%= yield %>
-      </main>
-    </div>
+    <% if show_account_layout %>
```

Comment:
Could the new markup be placed in a partial that's then included here? Appreciate that this could be a premature optimisation though.

Reply:
I've placed some of the markup in partials (it might be premature like you said, but it's much better for readability so I agreed it was worth doing!)